Examples/Mirrorator/README.md

```diff
@@ -0,0 +1,61 @@
+# iOS Mirrorator 
+
+Send data over Bluetooth to your Mirror templates!
```

Comment:
I think the intro could be a lil' more explanatory, basically saying that this is a tool/helper to quickly change templates and populate them with data, for more info read about custom templates here on the Dev Portal, etc.

Reply:
Sure, this will be the description then:

> Mirrorator is a prototyping tool for developing custom templates on Mirror. Switch between templats and populate them with data in a breeze! Develop a custom template for Mirror by following the instructions in [the Estimote Dev Portal](http://developer.estimote.com/mirror/build-your-own-template/)!